### PR TITLE
Ensure GL context as soon as the accelerated widget is available

### PR DESCRIPTION
**Pull request by chinmaygarde**
Doing this eagerly saves ~50ms on startup
screen shot 2015-12-03 at 2 01 34 pm

### CODE REVIEW

**Comment by jamesr** [comment]
Is it just a coincidence in that trace or does this operation have to complete before Engine::RunFrom... can start?

**Comment by chinmaygarde** [comment]
Thats just a coincidence

File: sky/shell/gpu/direct/rasterizer_direct.cc [hunk 1]
```diff
@@ -53,6 +53,7 @@ void RasterizerDirect::OnAcceleratedWidgetAvailable(gfx::AcceleratedWidget widge
   surface_ =
       gfx::GLSurface::CreateViewGLSurface(widget, gfx::SurfaceConfiguration());
   CHECK(surface_) << "GLSurface required.";
+  EnsureGLContext();
```
Comment (eseidelGoogle) [inline sky/shell/gpu/direct/rasterizer_direct.cc]: I might add a comment to explain why we do this eagerly.  Someone could find it via git blame of course.

**Comment by eseidelGoogle** [comment]
@abarth is your best reviewer, but lgtm.

**Comment by abarth** [comment]
LGTM w/ comment

**Comment by chinmaygarde** [comment]
Added comment. Rebased.

**Pull request merged**